src/components/Modals/BuySell/providers/GemManager.tsx

```diff
@@ -0,0 +1,194 @@
+import { ArrowBackIcon, CheckIcon, ChevronRightIcon, CopyIcon, ViewIcon } from '@chakra-ui/icons'
```

Comment:
Same comment about this living in `/providers`, this is not a context provider.
What about `ramp(s)` naming for everything currently named `provider(s)`?